### ISSUE TITLE
New syntax disk_encryption_key is not working

### ISSUE THREAD

**Issue by zheli** [terraform-providers/terraform-provider-google#2215]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment
* If an issue is assigned to the "modular-magician" user, it is either in the process of being autogenerated, or is planned to be autogenerated soon. If an issue is assigned to a user, that user is claiming responsibility for the issue. If an issue is assigned to "hashibot", a community member has claimed the issue already.

### Terraform Version
Terraform v0.11.8
+ provider.google v1.19.0

### Affected Resource(s)

* google_compute_disk

### Terraform Configuration Files

```tf
resource "google_compute_disk" "encrypted-disk" {
  name = "encrypted-disk"
  disk_encryption_key {
    raw_key = "oyFPUxmw1TWBIF1AkocHvEsqHldOy66Qkgc9t1LGIEg=" # base64 encoded 32 chars string
  }
}
```

### Debug Output

https://gist.github.com/zheli/a8de709797ab7d9d53dd328ea01ff6b8

### Expected Behavior

Disk encryption type is "Customer supplied".

### Actual Behavior

Disk encryption type is "Google-managed".

### Steps to Reproduce

1. `terraform apply`

### References

* #0000

**Comment by danawillow** [comment]
@chrisst, is this the same bug you found?

**Comment by chrisst** [comment]
Will be fixed in 2.0.0
